### PR TITLE
Removed Log reference from LocationPickerActivity 

### PR DESCRIPTION
**Pull request by ferranpons**
We still have Log reference that are being shown if Google Api fails. So we need to remove it and track them if necessary.

### CODE REVIEW

File: leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java [hunk 1]
```diff
@@ -723,7 +722,7 @@ private void startVoiceRecognitionActivity() {
       try {
         startActivityForResult(intent, REQUEST_PLACE_PICKER);
       } catch (ActivityNotFoundException e) {
-        Log.d(LocationPickerActivity.class.getName(), e.getMessage());
+        setTracking(TrackEvents.startVoiceRecognitionActivityFailed);
```
Comment (rocboronat) [inline leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java]: Just a suggestion: calling this method just `track` is... sexier! 🌶 

`setTracking` doesn't express the idea that it does something right now. But it's a totally ignorable comment :·D

Comment (ferranpons) [inline leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java]: @rocboronat You're right. `track` express better the intent of what it does. I'm going to change it. Thanks for the suggestion!

Comment (ferranpons) [inline leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java]: done!

Comment (rocboronat) [inline leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java]: Wow, you are welcome! Happy to see that you liked it! :tada:

**Pull request merged**